ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -205,6 +206,7 @@
 (defn metadata->concept
   "Create a metadata concept from the given metadata"
   [concept-type metadata content-type headers]
+  (info (format "Size of metadata to be ingested: [%s]" (memory-meter/measure metadata)))
```

Comment:
I don't think we care about how JVM allocate space for the metadata. We could just use JAVA's `metadata.getBytes().length` call to get the bytes of metadata.